### DISCUSSION TITLE
How to keep track of all non-computable validations done by the Java validator?

### DISCUSSION THREAD

**Discussion by ewoutkramer** [FirelyTeam/firely-net-sdk#2256] [Ideas]
I just read https://jira.hl7.org/browse/FHIR-24654 - it's definitely a nice validation to add to StructureDefinition, but it made me wonder: how are we going to ensure that we (eventually) get all of these validations into the .NET validator?  I am pretty sure the HL7-provided testset will not cover everything.  Maybe it is feasible to monitor the Java source for the validator - I hope these hand-written validation rules are kept somewhere at a central place?

Other ideas?

**Comment by alexzautke** [comment]
I’ve personally subscribed to the release notifications of the hl7.fhir.core repository. The describe accurately what has changed. Someone could look out for new validation changes here.

**Comment by ewoutkramer** [comment]
![image](https://user-images.githubusercontent.com/2276305/192575149-ac17b032-8454-41c1-8dc2-e32e6ff68299.png)

Additionally: The code in purple are the two functions that contain all "manual" validations in the Java validator.

**Comment by ewoutkramer** [comment]
See https://github.com/FirelyTeam/firely-validator-api/issues/266.
